src/main/java/seedu/duke/command/ListCommand.java

```diff
@@ -10,19 +12,31 @@ public ListCommand(String userInstruction) {
 
     @Override
     public void execute() {
-        System.out.println("Do " + userInstruction);
+        // System.out.println("Do " + userInstruction);
 
-        // Refactor into UI Class
         // Get Data
         NusModList modData = NusModList.sharedInstance();
+        // Check user Instruction, to refactor
+        if (userInstruction.equals("all")) {
+            // Refactor into UI Class
 
-        //List and Format Data - List 15 Modules tbc
-        System.out.println("Here are the list of available Modules (Limit to 10) : ");
-        for (int i = 0; i < 15; i++) {
-            System.out.println(modData.getMod(i).getModuleCode() + " | "
-                                + modData.getMod(i).getModuleTitle());
+            //List and Format Data - List 15 Modules tbc
+            System.out.println("Here are the list of available Modules (Limit to 10) : ");
+            for (int i = 0; i < 15; i++) {
+                System.out.println(modData.getMod(i).getModuleCode() + " | "
```

Comment:
Maybe we could throw this line to the ui class and call the function. I think we might reuse the function quite often